sample/src/main/kotlin/com/bnorm/react/index.kt

```diff
@@ -3,11 +3,16 @@
 package com.bnorm.react
 
 import kotlinx.browser.document
+import react.buildElement
 import react.dom.render
 import styled.injectGlobal
 import com.bnorm.react.IndexStyles as styles
 
 fun main() {
   injectGlobal(styles.global)
-  render(document.getElementById("root")) { App() }
+
+  val container = document.createElement("div")
+  document.body!!.appendChild(container)
+
+  render(buildElement { App() }, container)
 }
```

Comment:
Why the need to create a new element rather than using `root` as before? Wouldn't `render(buildElement { App() }, document.getElementById("root"))` work?

Reply:
I used the new syntax, but I reverted it to the legacy one in the latest commits